<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="MAVBCP733" name="com.internetapp.tests.ParasBankTest" tests="1" failures="0" timestamp="28 Jul 2020 14:01:34 GMT" time="23.886" errors="1">
  <testcase name="Test2" time="23.886" classname="com.internetapp.tests.ParasBankTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.maveric.core.testng.listeners.RetryAnalyzer.getCount(RetryAnalyzer.java:35)
	at com.maveric.core.testng.listeners.ReportListener.afterInvocation(ReportListener.java:70)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeAfterInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:100)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:566)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:713)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- Test2 -->
</testsuite> <!-- com.internetapp.tests.ParasBankTest -->
